.github/workflows/ci_tests_dev.yaml

```
@@ -17,7 +18,7 @@ on:
   # push:
   #   branches: [ main ]
   pull_request:
-    types: [ready_for_review]
+    types: [opened, reopened, synchronize, ready_for_review]
```

TODO revert this before merging.

```suggestion
    types: [ready_for_review]
```

---

Reverted in 8bda1a78d237e37f2b0b28873eda1e7161ec6149